module/index.js

```diff
@@ -106,6 +106,7 @@ import toLowerCase from './toLowerCase';
 import toUpperCase from './toUpperCase';
 import uncurry from './uncurry';
 import uncurry3 from './uncurry3';
+import unfold from './unfold';
```

Comment:
This want work with the named export. It would be strange for the user if this would be differently exported than the rest.
I think the only solution is to change the export to:
`export default function unfold...`